.github/workflows/canary.yml

```diff
@@ -0,0 +1,92 @@
+name: Canary
+on:
+  pull_request:
+    branches-ignore:
```

Comment:
I belive what we want is `branches`, so when any PR to master is opened, Canary is deployed

Reply:
Thanks @maksymivash, but what we're hoping for is canary's pre merge to `master`. The idea being that we would test our PR's in a production environment before releasing the official versions of the SDK. Which currently happens when we merge to master.

Reply:
Sure, but `branches-ignore` would do the opposite, all PRs created against not master branch would be deployed.

Reply:
In other words, do we want current PR to publish canary?

Reply:
So is this more of what we want?
```
pull_request:
     branches:
     - master
```

Reply:
> In other words, do we want current PR to publish canary?
Yes :-)

Reply:
@maksymivash @Ariflo what I was going for was that every PR that is opened, excluding PRs to master, would create a canary release.

I think this PR accomplishes that but let me know what everyone thinks.

Reply:
Yeah @bcleary06 sorry for the misunderstanding, I was under the impression that we wanted what Mack just broke down ☝️.

Reply:
@maksymivash @Ariflo you are both correct. I forgot the base branch was master and not development. I will update this.

Reply:
Nice, I just a deployment email request. Looks like it's working.